scene/main/scene_tree.cpp

```diff
@@ -701,6 +702,15 @@ void SceneTree::_notification(int p_notification) {
 
 		} break;
 
+		case NOTIFICATION_WM_ABOUT: {
+
+#ifdef TOOLS_ENABLED
+			EditorNode::get_singleton()->show_about();
```

Comment:
That won't work if receiving the notification while playing the game with a `tools=yes` binary. You should likely check the editor hint in the `#ifdef`, and if not also default to propagating the notification.